step-functions/step-functions-client-api/src/main/java/step/functions/execution/TokenLifecycleInterceptor.java

```diff
@@ -0,0 +1,7 @@
+package step.functions.execution;
+
+public interface TokenLifecycleInterceptor {
+    void onReturnTokenHandle(String tokenHandleId);
```

Comment:
Some listener might need the TokenWrapper (Quota by token type, etc). Instead of passing the Id I would therefore pass the token directly